### DISCUSSION TITLE
About Hss_cxx and Master Branch-  precondition failure

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#271] [General]
Hi @herlesupreeth   i just gone through latest changes today .

I compared master branch and hss_cxx branch commits

In master you fixed precondition failure issue . 
Same precondition failure issue also fixed in hss_cxx branch? Because i am not seeing same commit in hss_cxx branch

shall i use master or hss_cxx branch ?

**Comment by herlesupreeth** [comment]
Oops, i missed that. Thanks for letting me know. I will fix that tomorrow but you can cherry-pick that commit when using hss_cx_branch

**Reply by Sivanesh1992** [reply]
Thankyou @herlesupreeth   let me know once its done.

**Reply by herlesupreeth** [reply]
I have updated it now

**Reply by Sivanesh1992** [reply]
hi @herlesupreeth  Thankyou for the update
